api/app/Models/Community.php

```diff
@@ -63,4 +63,54 @@ public function roleAssignments(): HasManyThrough
         // There might be a better way to do this
         return $this->hasManyThrough(RoleAssignment::class, Team::class, 'teamable_id');
     }
+
+    /**
+     * Attach the users to the related team creating one if there isn't already
+     *
+     * @param string|array  $userId - Id of the user or users to attach the role to
+     * @return void
+     */
+    public function addCommunityRecruiter(string|array $userId)
+    {
+        $team = $this->team()->firstOrCreate([], [
+            'name' => "community-".$this->id,
+        ]);
+
+        if (is_array($userId)) {
+            foreach ($userId as $singleUserId) {
+                $user = User::find($singleUserId);
+                // $user->addRole('community_recruiter', $team->name);
```

Comment:
If you were waiting for https://github.com/GCTC-NTGC/gc-digital-talent/pull/10533 for the new roles to be available, its not merged into main.

Reply:
I've updated to use the roles in [8f3f742](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10621/commits/8f3f742c961de9a210e701b6d07fb8086f199877).